<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QDialog" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="digit0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setpoint">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeSign">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiply">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divide">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="power">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>POW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="factorial">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pi">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logarithm">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backspace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ClearAll</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineShow">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
